src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -679,6 +679,19 @@ public ResponseEntity<String> getShortCircuitAnalysisStatus(@Parameter(descripti
                 ResponseEntity.noContent().build();
     }
 
+    @PostMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/shortcircuit/result/csv")
+    @Operation(summary = "Get a short circuit analysis csv result")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The short circuit analysis csv export"),
+        @ApiResponse(responseCode = "204", description = "No short circuit analysis has been done yet"),
+        @ApiResponse(responseCode = "404", description = "The short circuit analysis has not been found")})
+    public byte[] getShortCircuitAnalysisCsvResult(
```

Comment:
ResponseEntity<byte[]> insteads of byte[]

Reply:
done